lib/views/loading_indicator/loading_indicator_utils.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:kayla_flutter_ic/views/loading_indicator/loading_indicator_view.dart';
+
+void showOrHideLoadingIndicator({
```

Comment:
@nmint8m We can define the extension for context instead
```suggestion
extension BuildContextExtension on BuildContext {
  showOrHideLoadingIndicator(bool shouldShow) {
  //...Do stuff things
  }
}
```